assets/js/blocks/cart-checkout/payment-methods/saved-payment-method-options.js

```diff
@@ -44,6 +44,17 @@ const getCcOrEcheckLabel = ( { method, expires } ) => {
  * @return {string} label
  */
 const getDefaultLabel = ( { method } ) => {
+	/* For saved payment methods with brand & last 4 */
+	if ( method.brand && method.last4 ) {
+		return sprintf(
+			/* translators: %1$s is referring to the payment method brand, %2$s is referring to the last 4 digits of the payment card. */
+			__( '%1$s ending in %2$s', 'woo-gutenberg-products-block' ),
+			method.brand,
+			method.last4
+		);
+	}
+
+	/* For saved payment methods without brand & last 4 */
```

Comment:
Thank you for this PR @achyuthajoy. I would have probably defaulted to have this logic inside `getCcOrEcheckLabel` and just generalizing that one if it make sense? so instead of having `getCcOrEcheckLabel` and `getDefaultLabel`, just have a single `getLabel` function that supports expires. This would allow payment methods that aren't CC or Echeck to define an expiry date.

Reply:
After discussing this in Slack, it seems the function handles more than just one case (it also handles things like not having a brand or a last 4), merging them in a single function would add further complication so let's just keep it as is.

Reply:
Thanks for the review @senadir. Is it good to merge?